### DISCUSSION TITLE
Welcome to Fusio Discussions!

### DISCUSSION THREAD

**Discussion by chriskapp** [apioo/fusio#532] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marifrahman** [comment]
Hi @chriskapp , 
First thanks a lot for this great tool :) I am atm exploring the tool and trying to understand bits a pieces.
However, I got a got quick question (posted in the Q&A section) - basically how to do POST Operation with some data in the Body so that http-Processor action can POST with the same body parameter ? I can check this from Action Designer - however, not sure how to do it from Operation and then call the external api.
Again much appreciated for this great tool.

**Reply by chriskapp** [reply]
Hi @marifrahman thanks for the feedback, I have just tried to answer you question, let me know whether this works as expected 👍
